### DISCUSSION TITLE
How to read in Pivot Points

### DISCUSSION THREAD

**Discussion by Rhidian12** [assimp/assimp#5336] [Q&A]
Hi!

I'm working on reading in .fbx files with assimp, and I'm reading in 3 cubes from an assimp file, and one thing I noticed is that the pivot is not being taken into account.

![image](https://github.com/assimp/assimp/assets/74491212/dd792347-061c-4b42-81ef-0143a449a51b)

These cubes are all at Y=0, but in the .fbx file, they're all aligned at the _bottom_, not in the middle.
Can I somehow get the pivot information from .fbx files using assimp?

Thanks a lot!

**Comment by kimkulling** [comment]
At first, you need to set the property **AI_CONFIG_IMPORT_FBX_PRESERVE_PIVOTS** to true. 
Because Assimp currently does not support any pivot points this feature will be part of the calculated transformation chain which is represented in the node hierarchy.

Assimp seems to need a lot more features.
